ksm/ksm_day13/키패드 누르기.js

```diff
@@ -0,0 +1,47 @@
+function solution(numbers, hand) {
+    // 초기 손 위치: *을 10으로, #을 12로 가정
+    const handPosition = {left: 10, right: 12}; 
+    
+    const answer = numbers.reduce((acc, cur) => {
+        if (cur === 0) cur = 11; // 0을 11로 변환
+        
+        const remainder = cur % 3; // 열 구분: 3으로 나눈 나머지
+
+        // 나머지 1 (왼쪽 열 1, 4, 7)
+        if (remainder === 1) {
+            acc.push("L"); // 왼손으로 처리하고
+            handPosition.left = cur; // 왼손 위치 갱신
+        }
+        // 나머지 0 (오른쪽 열: 3, 6, 9)
+        else if (remainder === 0) {
+            acc.push("R"); //오른손으로 처리하고
+            handPosition.right = cur; // 오른손 위치 갱신
+        }
+        // 나머지 2 (가운데 열: 2, 5, 8, 0)
+        else {
+            const x = (num) => Math.floor((num - 1) / 3); // 행 계산 함수
+            const y = (num) => (Math.floor((num - 1)) % 3); // 열 계산 함수
+
+            // 현재 숫자에서 왼손, 오른손 위치 간의 맨해튼 거리 계산
+            const left = Math.abs(x(cur) - x(handPosition.left)) + Math.abs(y(cur) - y(handPosition.left));
+            const right = Math.abs(x(cur) - x(handPosition.right)) + Math.abs(y(cur) - y(handPosition.right));
```

Comment:
이런걸 맨해튼 거리 계산이라고 하는군요 ~ 하나 배워갑니다
```suggestion
            const right = Math.abs( x(cur) - x(handPosition.right) ) + Math.abs( y(cur) - y(handPosition.right) );
```
중첩해서 함수 호출하실 때 공백이 한 칸 있으면 좋겠습니다 ㅎ
추가로 행, 열 계산 함수의 이름을 명확히 바꿔보는건 어떠신가요?

Reply:
리뷰 감사합니다! 
개인적으로 공백을 넣음으로써 괄호의 시작과 끝이 더 명확하게 구분되는 것 같아서 조금 더 가독성이 좋은 것 같습니다.🙂‍↕️
그리고 행, 열 계산 함수 이름을 조금 더 명확한 함수 이름으로 수정하였습니다~